client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -17,6 +17,7 @@ import { CHECKLIST_KNOWN_TASKS } from 'calypso/state/data-layer/wpcom/checklist/
 import { requestGuidedTour } from 'calypso/state/guided-tours/actions';
 import getChecklistTaskUrls from 'calypso/state/selectors/get-checklist-task-urls';
 import getSiteChecklist from 'calypso/state/selectors/get-site-checklist';
+import isSiteUsingCoreSiteEditor from 'calypso/state/selectors/is-site-using-core-site-editor.js';
```

Comment:
We're trying to get rid of this selector! (https://github.com/Automattic/wp-calypso/issues/57625)

Can you use the equivalent from the /sites/block-editor endpoint, instead? Here's an example: https://github.com/Automattic/wp-calypso/pull/57626/files#diff-86275381173bf7a239b20c0a15895dde9ac3dd241464cd47b3e670d12ed0f072

I think `is_fse_active` is what we should use here.

Reply:
Done